docs/nightly/en/greptimecloud/migrate-to-greptimecloud/migrate-from-influxdb.md

```diff
@@ -0,0 +1,184 @@
+---
+template: ../../db-cloud-shared/migrate/migrate-from-influxdb.md
+---
+# Migrate from InfluxDB
+
+<docs-template>
+
+{template get-database-connection-information%
+
+Navigate to the [GreptimeCloud console](https://greptime.cloud) and click the `Connection Information` section under `Manage Your Data`.
+You can find the GreptimeDB URL, database name, as well as the username and password associated with the token.
+
+%}
+
+{template write-data-http-api%
+::: code-group
+
+```shell [InfluxDB line protocol v2]
+curl -X POST 'https://<host>/v1/influxdb/api/v2/write?db=<db-name>' \
```

Comment:
```suggestion
curl -X POST 'https://<host>/v1/influxdb/api/v2/write?bucket=<db-name>' \
```

A more InfluxDB native URL use `bucket` param. We alias `bucket` to `db` in our implementation, but it's not what InfluxDB v2 defines.

Reply:
Ditto others.